.github/workflows/Build-Test-And-Deploy.yml

```diff
@@ -99,19 +99,19 @@ jobs:
           docker image ls -a
 
       - name: Log in to container registry
-        if: github.event_name != 'pull_request_target'
+        if: github.event_name != 'pull_request_target' && github.event_name != 'pull_request'
```

Comment:
You shouldn't need this if since you are doing the same check at the job level, it wont get down to these steps.

Reply:
Fixed